docs/source/specs/openapi.yml

```diff
@@ -0,0 +1,84 @@
+openapi: "3.0.0"
```

Comment:
I'm pretty swagger/openAPI ignorant. What's generating this YAML?

Reply:
Nothing is generating this YAML.

I'm actually in favor of this being a part of "documentation-driven development". You can have an issue to define the API contract for the UI to use and they are then able to use a mock of that API while the implementation is developed. QE can also use this doc to write test cases while development is in progress (ideally).

Reply:
Yep, totally into it. Creating stubs to unblock is 👍 . 

The root of my question was around that contract. If the swagger YAML always comes first, then there's way less risk of forgetting to update it with API changes.